internal/fdo.go

```diff
@@ -64,9 +64,57 @@ func fdoLookupApplication(theme string, size int, appName string) desktop.IconDa
 			return newFdoIconData(theme, size, testLocation)
 		}
 	}
+	//No Match Was Found Checking File Names - Check Files Themselves
```

Comment:
This comment is slightly ambiguous. You mean "file contents" right?
When a comment has to be written it indicates a new function with a clear name could be used.

How about "fdoLookupApplicationByMetadata" to fit with your current naming?